src/components/implementation/tests/micro_booter/micro_booter.c

```diff
@@ -36,9 +36,11 @@ printc(char *fmt, ...)
 #define PRINT_FN prints
 #define debug_print(str) (PRINT_FN(str __FILE__ ":" STR(__LINE__) ".\n"))
 #define BUG() do { debug_print("BUG @ "); *((int *)0) = 0; } while (0);
+#define BUG_DIVZERO() do { debug_print("BUG @ "); int i = 1 / 0; } while (0);
 
 struct cos_compinfo booter_info;
 
+#define ITERATIONS 1000000
```

Comment:
No change necessary here.  However, keep in mind in the future that if you're going to be dividing, it is always beneficial to do so with a power of two (so that you can use shifts instead of divide instructions).  1<<20 would work here.  That said, we don't care about the performance of these divides, so it doesn't matter.  However, this is a good habit to get into.